Sprint-3/2-mandatory-rewrite/2-is-proper-fraction.js

```diff
@@ -1,6 +1,9 @@
 function isProperFraction(numerator, denominator) {
```

Comment:
Within your else statement, it can be simplified. 
```
    if (numerator < denominator) {
      return true;
    } else return false; 
```

Even more simple is this:
```
return numerator < denominator
```
This will return true if the numerator is less than the denominator. It will return false for everything else. 

If you need to check against having a denominator of zero, you can add this check to your function

Reply:
Did you implement the change? I don't see this in the codebase. Perhaps you forgot to commit your changes?